node/pkg/accountant/submit_obs.go

```diff
@@ -237,22 +259,22 @@ func (acct *Accountant) submitObservationsToContract(msgs []*common.MessagePubli
 		status, exists := responses[msgId]
 		if !exists {
 			// This will get retried next audit interval.
-			acct.logger.Error("did not receive an observation response for message", zap.String("msgId", msgId))
+			acct.logger.Error(fmt.Sprintf("did not receive an observation response from %s for message", tag), zap.String("msgId", msgId))
 			submitFailures.Inc()
 			continue
 		}
 
 		switch status.Type {
 		case "pending":
-			acct.logger.Info("transfer is pending", zap.String("msgId", msgId))
+			acct.logger.Info(fmt.Sprintf("transfer is pending on %s", tag), zap.String("msgId", msgId))
```

Comment:
Just out of curiosity, why use fmt instead of a zap.String?

Reply:
Because this way the log message text can say "transfer is pending for accountant" or "transfer is pending for ntt-accountant". That way there will not be unnecessary references to "ntt" in the logs (like if I did zap.Bool("isNTT", isNTT)).